src/charm.py

```diff
@@ -19,13 +19,17 @@
 from ops.charm import CharmBase, CharmEvents
 from ops.framework import EventBase, EventSource, StoredState
 from ops.main import main
-from ops.model import ActiveStatus, BlockedStatus, WaitingStatus
+from ops.model import ActiveStatus, BlockedStatus, ErrorStatus, WaitingStatus
 from slurm_ops_manager import SlurmManager
 from utils.manager import SlurmdbdManager
 
 logger = logging.getLogger(__name__)
 
 
+SLURM_ACCT_DB = "slurm_acct_db"
```

Comment:
Why did you make the slurmdbd accounting database name a constant? Do you think this is value that administrators will want to change, or do you think we could just reference the string directly?

Reply:
It is a constant. We don't expect this to change, so might as well make it a constant and treat it as such. Keeping it as a constant - the tests and python tool chain complains when its not correct. When its just a string that is fat-fingered elsewhere, its more difficult and requires runtime checks to confirm.